plugins/VaccineAppointmentScanner/__init__.py

```diff
@@ -0,0 +1,119 @@
+from nextcord.ext import commands, tasks
+from Plugin import AutomataPlugin
+
+import requests
```

Comment:
Currently the projects moved away from requests -> httpx if you can refactor for that, that'd be great. But if not possible, probably no big deal. This could help you get started for your case https://www.python-httpx.org/advanced/

Reply:
The requests library has been swapped for httpx, it's pretty much the same as before except there's a `with` block now.